config/cscs.py

```diff
@@ -99,6 +99,42 @@ class ReframeSettings:
                 }
             },
 
+            'fulen': {
+                'descr': 'Fulen',
+                'hostnames': ['fulen'],
+                'modules_system': 'tmod',
+                'resourcesdir': '/apps/common/UES/reframe/resources',
+                'partitions': {
+                    'login': {
+                        'scheduler': 'local',
+                        'environs': ['PrgEnv-gnu'],
+                        'descr': 'Login nodes',
+                        'max_jobs': 1
+                    },
+
+                    'normal': {
+                        'scheduler': 'nativeslurm',
+                        'environs': ['PrgEnv-gnu'],
+                        'descr': ('Compute nodes - '
+                                  'default partition'),
+                    },
+
+                    'fat': {
+                        'scheduler': 'nativeslurm',
+                        'environs': ['PrgEnv-gnu'],
+                        'access':  ['--partition fat'],
```

Comment:
There is a redundant space here. Either align it to the above value or remove the space.

Reply:
Same in other places.